syntax = "proto3";

message Class {
  string class_id = 1;
  string subject_id = 2;
  int32 credit = 3;
  string teacher_public_key = 4;
  string edu_officer_public_key = 5;
  string institution_public_key = 6;
  uint64 timestamp = 7;
  string transaction_id = 8;
}

message ClassContainer {
  repeated Class entries = 1;
}